source/1_engine/rex_engine/src/string/stringentry.cpp

```diff
@@ -0,0 +1,97 @@
+#include "rex_engine/string/stringentry.h"
+
+#include <rex_std/memory.h>
+
+namespace rex
+{
+  //-------------------------------------------------------------------------
+  StringEntry::StringEntry(rsl::string_view characters) noexcept
+      : m_characters()
+  {
+    m_characters.reset(static_cast<char*>(malloc(characters.size() + 1))); // NOLINT(cppcoreguidelines-no-malloc)
```

Comment:
don't use malloc, the memory won't get tracked if you do.
just a note, you can use rsl::make_unique<char[]>(characters.size() + 1); here
that'll use operator new internally and will track memory.
cppcoreguidelines-no-malloc should never be ignored unless approved by both you and me

Reply:
Okay, thanks!